### ISSUE TITLE
[Bug]: Qwen2-VL-7B with sglang (vLLM-back) Performance Degradation on MME benchmark

### ISSUE THREAD

**Issue by thusinh1969** [vllm-project/vllm#10588]
### Your current environment

This issue has been raised here. Please have a look.

https://github.com/sgl-project/sglang/issues/2112

### Model Input Dumps

This issue has been raised here. Please have a look.

https://github.com/sgl-project/sglang/issues/2112

### 🐛 Describe the bug

This issue has been raised here. Please have a look.

https://github.com/sgl-project/sglang/issues/2112

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by DarkLight1337** [comment]
Might be related to #9825

**Comment by yileld** [comment]
> Might be related to #9825

No, this problem is diffrent from this issue. It's about the discrepancies in the calculation of the logits in the vision head.

**Comment by DarkLight1337** [comment]
Do you still get similar problems on the latest released version of vLLM?
